react/week1/my-react-app/src/Todo.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "./todo.css";
+
+const Todo = ({id,description,deadlineDate}) =>{
+    return (
+    <div className="todoitem">
+        <h2 className="id">{id}</h2>
```

Comment:
Following my suggestion your code would be like that:
```suggestion
        <h2 className="id">{todoItem.id}</h2>
```
and so on..